ftml/docs/Philosophy.md

```diff
@@ -0,0 +1,36 @@
+[<< Return to the README](../README.md)
+
+## Philosophy
+
+Wikitext is similar to Markdown and dissimilar to C in that the grammar is loose.
+Any invalid token combinations are rendered as-is, rather than producing a fatal parsing
+error which halts the process. This latter, C-like (or any programming language, really)
+philosophy was how the original version of ftml operated. However this presents obvious
+incomaptibilities with Wikidot, and the grammar had to be increasingly complicated to handle
+edge-case conditions.
+
+This rewrite of ftml performs preprocessing substitions and tokenization like the first version,
+but the parser is hand-written to allow for loose fallback rules.
```

Comment:
I'm not sure the distinction between the current version of ftml and the original version, while historically important in terms of code history and developer experience, important reading for what I expect to be the average reader of this document. I feel like the philosophy of fallbacks could be described with the assumption that the current version is the only canonical version